scim/resource_user.go

```diff
@@ -44,6 +43,14 @@ func ResourceUser() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			}
+			m["delete_repos"] = &schema.Schema{
```

Comment:
perhaps doing `force_delete_content` is a better setting name for both repos and notebooks.

Reply:
And only include one flag for deleting both repos and dirs?

Reply:
Changed so that there's one `force_delete_content ` flag and downstream if statement.